receiver/opencensusreceiver/metadata.yaml

```diff
@@ -5,3 +5,5 @@ status:
   stability:
     beta: [metrics, traces]
   distributions: [core, contrib, observiq, sumo, redhat]
+  codeowners:
+    active: [open-telemetry/collector-approvers]
```

Comment:
This

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS#L218C10-L218C28
```
receiver/opencensusreceiver/                             @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
```